src/renderer/index.tsx

```diff
@@ -5,12 +5,25 @@ import './services/i18n';
 import { isDevelopment } from '../common/env-and-consts';
 
 import AppLoadFallback from './components/app-load-fallback';
+import { RETAINED_LOCALSTORAGE_KEYS } from './utils/localstorage-clear';
 import './index.scss';
 
 if (!isDevelopment) {
   document.body.style.overflow = 'hidden';
 }
 
+const clearStorage = () => {
```

Comment:
This belongs in the `utils`, and name it more specifically - this does not clear storage, it also backs up values. Also it's not clearing any storage. It's clearing local storage